src/targets/github.ts

```diff
@@ -345,6 +337,7 @@ export class GitHubTarget extends BaseTarget {
     await this.github.repos.updateRelease({
       ...this.githubConfig,
       release_id: release.id,
+      make_latest: isLatest ? 'true' : 'false',
```

Comment:
Why is this a string?

Reply:
Here: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release it is documented as type `string`, for whatever reason...